aiogram/types/message.py

```diff
@@ -1567,29 +1567,36 @@ def link(self, text, as_html=True) -> str:
     async def send_copy(
             self: Message,
             chat_id: typing.Union[str, int],
-            with_markup: bool = False,
             disable_notification: typing.Optional[bool] = None,
             reply_to_message_id: typing.Optional[int] = None,
+            reply_markup: typing.Union[InlineKeyboardMarkup, ReplyKeyboardMarkup, None] = None,
+            parse_mode: typing.Union[base.String, None] = None,
     ) -> Message:
         """
         Send copy of current message
 
         :param chat_id:
-        :param with_markup:
         :param disable_notification:
         :param reply_to_message_id:
+        :param reply_markup:
+        :param parse_mode:
         :return:
         """
-        kwargs = {"chat_id": chat_id, "parse_mode": ParseMode.HTML}
+        kwargs = {"chat_id": chat_id, "reply_markup": self.reply_markup}
```

Comment:
Can we use `{"reply_markup": reply_markup or self.reply_markup}`?

Reply:
Hmm, I think this is better than using if

Reply:
> Can we use `{"reply_markup": reply_markup or self.reply_markup}`?

Hmm, I think this is better than using if